### DISCUSSION TITLE
Add new referent: methodology

### DISCUSSION THREAD

**Discussion by caouecs** [Laravel-Lang/common#19] [Q&A]
To add a new referent, we do a poll with all team ?

**Comment by olsza** [comment]
I added a link in the auto-add topic (https://github.com/Laravel-Lang/lang/discussions/1389#discussioncomment-93858)

unless this topic is supposed to be enough for all surveys maybe you can copy the survey link here ;)

**Comment by andrey-helldar** [comment]
I have not yet seen the whole team take part in any action. Often this is simply not needed.
If a new referent is accepted, I think this way: if it is clear that a person is sending well-designed PRs and is adequate in communication, then one or two votes will be enough for adding him to the team. Of course, if the person himself does not mind.

**Reply by andrey-helldar** [reply]
For example, I enjoyed working with @olsza and @ChristofK has some pretty good code.

**Reply by olsza** [reply]
thx

**Comment by caouecs** [comment]
And for @masterix21 ?

**Reply by andrey-helldar** [reply]
Now the Italian language is missing the translation of only two keys that were recently added to Laravel.

Basically, if he doesn't mind, he can join the team.

![image](https://user-images.githubusercontent.com/10347617/96909842-41be0c00-14a7-11eb-93b0-96a91845eb6e.png)


![image](https://user-images.githubusercontent.com/10347617/96909684-06234200-14a7-11eb-9123-846506155d88.png)

**Reply by masterix21** [reply]
Hi @andrey-helldar, please check PR #1435

Thanks

**Reply by andrey-helldar** [reply]
Approved

**Reply by caouecs** [reply]
ok to add him :)

**Reply by masterix21** [reply]
Updated italian translation #1459

**Comment by caouecs** [comment]
Hello all,
are you ok to add [keljtanoski](https://github.com/keljtanoski) as referent for mk language ?

https://github.com/Laravel-Lang/lang/pull/1521

**Reply by masterix21** [reply]
Yes, ok.

**Reply by andrey-helldar** [reply]
je suis pour

**Comment by andrey-helldar** [comment]
Lately @sotten has been helping us with the German translation. @sotten, would you like to become an referent?

@Laravel-Lang/laravel-lang, what do you think?

**Reply by sotten** [reply]
I can be gladly a referent for de, possibly also for questions in de_ch (very similar).

**Reply by andrey-helldar** [reply]
PR: https://github.com/Laravel-Lang/lang/pull/1765

@caouecs, can you add @sotten to our [team](https://github.com/orgs/Laravel-Lang/people)?

**Reply by caouecs** [reply]
done

**Reply by andrey-helldar** [reply]
@sotten, welcome! 🎉🎉🎉

**Reply by sotten** [reply]
@caouecs thank you for the invitation!

**Comment by andrey-helldar** [comment]
@EuCarlos, do you want to join our team and be responsible for translations pt_BR?

What do we have to do:
- Periodically check the availability of new keys for transfers;
- Translate missing keys;
- Carry out a code review of those who send the translation to pt_BR;
- Add your nickname to the [referents](https://laravel-lang.com/referents.html) page and call when PR appears.

@Laravel-Lang/laravel-lang, what do you think?

PRs:
- https://github.com/Laravel-Lang/lang/pull/2000
- https://github.com/Laravel-Lang/lang/pull/2002

**Reply by caouecs** [reply]
We add him ?

**Reply by EuCarlos** [reply]
> We add him ?

Not yet

**Reply by EuCarlos** [reply]
Of course I want to join the team

**Reply by andrey-helldar** [reply]
Oh! How did we miss this moment?

**Comment by andrey-helldar** [comment]
@Laravel-Lang/laravel-lang, hey!

I propose to accept a new member to our team - @ariaieboy.

He will oversee the translation into Persian.

@ariaieboy agree to join our team: 
 https://github.com/Laravel-Lang/lang/pull/2114#issuecomment-1092315343

Merged PR's:
- https://github.com/Laravel-Lang/lang/pull/2114
- https://github.com/Laravel-Lang/lang/pull/2110

What do you think?

**Reply by masterix21** [reply]
👍🏻

**Reply by caouecs** [reply]
ok for me

**Reply by andrey-helldar** [reply]
@caouecs, can you add him to the team?

**Reply by caouecs** [reply]
done

**Reply by andrey-helldar** [reply]
ty

**Reply by andrey-helldar** [reply]
@ariaieboy, welcome to the team 🤝

**Reply by mohamedsabil83** [reply]
agree, welcome to the team @ariaieboy

**Comment by andrey-helldar** [comment]
@Laravel-Lang/laravel-lang, hey!

I propose to accept a new member to our team - @EuCarlos.

He will oversee the translation into `pt_BR`.

@EuCarlos agree to join our team:

- https://github.com/Laravel-Lang/lang/pull/2122#issuecomment-1093456420
- https://github.com/Laravel-Lang/lang/pull/2104
- https://github.com/Laravel-Lang/lang/pull/2028

What do you think?

**Reply by mohamedsabil83** [reply]
Agree. it's good to have an active member(s) for every locale in our team.

**Reply by luisprmat** [reply]
I'm agree, welcome @EuCarlos to our team.

**Reply by caouecs** [reply]
done

**Reply by EuCarlos** [reply]
Thanks

**Reply by andrey-helldar** [reply]
@EuCarlos, you should receive an invitation link to join the team by email.

**Comment by andrey-helldar** [comment]
@olsza, hi! Have you left the team?
